src/css/base/_typography.scss

```
@@ -38,7 +38,7 @@ blockquote {
 
   &::before {
     color: var(--color--accent);
-    content: "\201C";
+    content: '\201C';
```

This should still be a double quote
```suggestion
    content: "\201C";
```